doc/topics/development/git/index.rst

```diff
@@ -32,7 +32,7 @@ Feature Release Branching
 
 Each feature release is maintained in a dedicated git branch derived from the
 last applicable release commit on develop. All file changes relevant to the
-feature release will be completed in the develop branch prior to the creation
+feature release will be completed in the ``master`` branch prior to the creation
 of the feature release branch. The feature release branch will be named after
```

Comment:
release tag instead of branch

Reply:
When it comes to releasing with tags instead of branches, how are we doing point releases / cve-fixes / etc? Does any branching occur at a past commit in order to make changes to a previous release?

Here is the text I'm unsure of how to re-word:

```rst
Release Tagging
===============

Salt project maintains two types of releases, ``Feature Releases`` and
``Point Releases`` (also commonly referred to as ``Bugfix Releases``. A
feature release is managed by incrementing the first or second release point
number, so 2015.5.5 -> 2015.8.0 signifies a feature release
and 2015.8.0 -> 2015.8.1 signifies a point release.

Feature Release Tagging
-----------------------

Each feature release is maintained in a dedicated git branch derived from the
last applicable release commit on develop. All file changes relevant to the
feature release will be completed in the ``master`` branch prior to the creation
of the feature release branch. The feature release branch will be named after
the relevant numbers to the feature release, which constitute the first two
numbers. This means that the release branch for the 2015.8.0 series is named
2015.8.

A feature release branch is created with the following command:

.. code-block:: bash

    # git checkout -b 2015.8 # From the master branch
    # git push origin 2015.8

Point Releases
--------------

As documented in `SEP 14 <https://github.com/saltstack/salt-enhancement-proposals/blob/master/accepted/0014-dev-overhaul.md#hotfix--patch-release>`__,
point releases should be rare.
```

Reply:
I am good to review this entire piece in another PR and leave it as-is here.